src/commands/android/subcommands/common.ts

```diff
@@ -0,0 +1,52 @@
+import colors from 'ansi-colors';
+import ADB from 'appium-adb';
+
+import {Options} from '../interfaces';
+import Logger from '../../../logger';
```

Comment:
I meant the opposite of this. Put outermost relative imports at the top and innermost at the bottom. And for collisions like multiple imports from the same level, follow the alphabetical order.

For example, if we have `../common` and `../utils/adb`, since both have equal numbers of `../`, we would place them in the alphabetical order of the first name after `../`, i.e., comparing `command` and `utils` in alphabetical order.

Let's keep fixing the imports in whichever file we touch.